ivy/array/general.py

```diff
@@ -56,3 +56,26 @@ def gather_nd(
         ivy.array(2)
         """
         return ivy.gather_nd(self, indices, out=out)
+
+    def to_numpy(self: ivy.Array):
+        """
+        ivy.Container instance method variant of ivy.to_numpy.
+        This method simply wraps the function, and so the docstring for
+        ivy.to_numpy also applies to this method with minimal changes.
+
+        Examples
```

Comment:
The docstrings test is failing because of this example:
```
output for  to_numpy  on run:  [1011][[1.0.1.1.][1.1.0.0.]] 
output in docs : [1011]>>>x=ivy.array([[1,0,1,1],[1.,1.,0.,0.]])>>>y=x.to_numpy()>>>print(y)[[1.0.1.1.][1.1.0.0.]][[1.0.1.1.][1.1.0.0.]]
```